### PR TITLE
Updated User guide: UG Walkthrough and Bug Warning 

### PR DESCRIPTION
**Pull request by AJZ1995**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -75,6 +75,41 @@ You can use `entrylist` and `orderlist` to switch between the two lists.
 
 Person List Panel, Center Panel and Right Panel will be explained in greater detail in <<Features>>.
 
+=== Walkthrough
+Here's a simple walkthrough of Contact Sails.
```
Comment (amad-person) [inline docs/UserGuide.adoc]: This is a good section, but the rest of us have given walkthroughs in the Features Section. Perhaps you can shift this to the respective command sections?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -75,6 +75,41 @@ You can use `entrylist` and `orderlist` to switch between the two lists.
...
+
+2) Let us start by clearing these example contacts. Type `clear` on the command box at the top.
+Now, you should see an empty screen like the picture below:
+
+image::Empty.png[width = "790"]
+
+3) Let us add our first contact now. Type `add n/Mary p/96628855 e/mary@example.com a/Mary Road, block 1, #12-06, g/SoC pr/videoGame`
+in the command box and press enter. Click on the contact Mary and you should see this picture:
+
+image::ExampleMary.png[width = "790"]
```
Comment (amad-person) [inline docs/UserGuide.adoc]: The pictures in the `images` folder have the `.PNG` extension, so the pictures won't show up in the UserGuide unless the extensions here are updated.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (AJZ1995) [reply docs/UserGuide.adoc]: To my understanding, walkthrough in features section are for features. This is a general walkthrough at the start, before the features section.

**Comment by AJZ1995** [comment]
Removed walkthrough and added some pictures for 2 parts in Section 4.2

**Pull request merged**